app/scripts/lib/constants.js

```diff
@@ -40,7 +40,8 @@ define(function (require, exports, module) {
     IFRAME_CONTEXT: 'iframe',
 
     SYNC_SERVICE: 'sync',
-
+    SYNC11: 'sync11',
```

Comment:
Can you add a `_MIGRATION` suffix to each of these so the intent is crystal clear?